assets/src/stories-editor/components/block-mover/draggable.js

```diff
@@ -61,13 +69,35 @@ class Draggable extends Component {
 	 * @param {Object} event The non-custom DragEvent.
 	 */
 	onDragEnd = ( event ) => {
-		const { onDragEnd = noop } = this.props;
+		const { onNeighborDrop, onNeighborHover, blockName, setTimeout, onDragEnd = noop } = this.props;
 		if ( event ) {
 			event.preventDefault();
 		}
 
+		// Make sure to reset hover
+		onNeighborHover( 0, true );
+
+		// Attempt drop on neighbor if offset
+		if ( this.pageOffset !== 0 ) {
+			// All this is about calculating the position of the (correct) element on the new page.
+			const currentElementTop = parseInt( this.cloneWrapper.style.top );
+			const currentElementLeft = parseInt( this.cloneWrapper.style.left );
+			const newLeft = currentElementLeft - ( this.pageOffset * PAGE_AND_MARGIN );
+			const blockIsCTA = isCTABlock( blockName );
+			const baseHeight = blockIsCTA ? STORY_PAGE_INNER_HEIGHT / 5 : STORY_PAGE_INNER_HEIGHT;
```

Comment:
Instead of 3 `blockIsCTA` statement, why not why if statement?

Reply:
Then you couldn't use `const`, no?

Reply:
They would change to lets yes.

Reply:
I reorganised the code in 1cde15c.